scripts/build.sh

```diff
@@ -19,4 +19,12 @@ export CGO_ENABLED=0
 
 # Build binary
 cd $GOPATH/src/github.com/raintank/metrictank
-go build -ldflags "-X main.GitHash=$GITVERSION" -o $BUILDDIR/metrictank
+
+OUTPUT=$BUILDDIR/metrictank
+
+if [ "$METRICTANK_BUILD" == "race" ]
```

Comment:
this env var is a bit long. what's wrong with just checking if `$1` is `-race` ? that seems simpler.